src/libs/Environment/Environment.js

```diff
@@ -43,7 +37,12 @@ function isInternalTestBuild() {
  */
 function getEnvironmentURL() {
     return new Promise((resolve) => {
-        getEnvironment().then((environment) => resolve(ENVIRONMENT_URLS[environment]));
+        getEnvironment().then((environment) => 
+        {
+            // Our approach is to use staging url only for staging environment and use production url for all other environments.
+            const environmentURL = (environment === CONST.ENVIRONMENT.STAGING) ?  Url.addTrailingForwardSlash(CONST.STAGING_NEW_EXPENSIFY_URL) : CONST.NEW_EXPENSIFY_URL;
```

Comment:
Again I am confused was't the staging URL being returned when the environment was STAGING? I mean the ENVIRONMENT_URLS ? I don't see a point of adding this change? @rojiphil 

@AndrewGable wdyt?